code/modules/shuttle/super_cruise/orbital_poi_generator/objective_types/vip_extraction.dm

```diff
@@ -6,11 +6,23 @@
 	max_payout = 200000
 
 /datum/orbital_objective/vip_recovery/get_text()
-	return "Someone of particular interest to use is located at [station_name]. We require them to be extracted immediately. \
-		We have good intel to suggest that the VIP is still alive, however if not their personal diary-disk should have enough infomation \
-		about what we are looking for. An additional point to note is that it is recommended a security team assists in this mission due \
+	return "Someone of particular interest to us is located at [station_name]. We require them to be extracted immediately. \
+		We have good intel to suggest that the VIP is still alive, however, if not, their personal diary-disk should have enough infomation \
+		about what we are looking for. In addition, it is recommended a security team assists in this mission due \
 		to the potentially hostile nature of the individual. Return the individual to the station alive to complete the objective."
 
+/datum/orbital_objective/vip_recovery/dangerous //You'll probably want to take one or two Officers along.
+	name = "VIP Recovery (DANGEROUS)"
+	min_payout = 30000
+	max_payout = 40000 //Needs to have a beefy paycheck attached to it so the explorers think it's worth it.
```

Comment:
This is considerably less btw, the VIP objective payout is 6 figures

Reply:
Mistake